packages/nimble-components/src/theme-provider/tests/theme-provider.stories.ts

```diff
@@ -0,0 +1,117 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import { tableTag } from '../../table';
+import { tableColumnDateTextTag } from '../../table-column/date-text';
+import {
+    sharedTableArgTypes,
+    type SharedTableArgs,
+    sharedTableArgs
+} from '../../table-column/base/tests/table-column-stories-utils';
+import { Theme } from '../types';
+import { Direction, themeProviderTag } from '..';
+
+const simpleData = [
+    {
+        date: new Date(1984, 4, 12, 14, 34, 19, 377).valueOf()
+    },
+    {
+        date: new Date(1984, 2, 19, 7, 6, 48, 584).valueOf()
+    },
+    {
+        date: new Date(2013, 3, 1, 20, 4, 37, 975).valueOf()
+    },
+    {
+        date: new Date(2022, 0, 12, 20, 4, 37, 975).valueOf()
+    }
+];
+
+const metadata: Meta = {
+    title: 'Tokens/Theme Provider',
+    parameters: {
+        docs: {
+            description: {
+                component: ''
+            }
+        }
+    },
+    argTypes: {
+        ...sharedTableArgTypes,
+        selectionMode: {
+            table: {
+                disable: true
+            }
+        }
+    },
+    args: {
+        ...sharedTableArgs(simpleData)
+    }
+};
+
+export default metadata;
+
+const langDescription = `Defines the language of the element. See [external documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) for details.
+
+Users should set \`lang\` on the root \`html\` element of the page to reflect the language of the content. If necessary, users may override the language for a subtree by inserting a \`nimble-theme-provider\` element and setting its \`lang\` attribute. Nimble elements will not honor a \`lang\` value set on any other type of ancestor element.`;
```

Comment:
nit: distinguishing between end-users and developers
```suggestion
Applications should set \`lang\` on the root \`html\` element of the page to reflect the language of the content. If necessary, users may override the language for a subtree by inserting a \`nimble-theme-provider\` element and setting its \`lang\` attribute. Nimble elements will not honor a \`lang\` value set on any other type of ancestor element.`;
```